docs/specification/1.0.md

```diff
@@ -464,6 +464,23 @@ Field | Optionality | Type | Description
 <nobr>`label`</nobr>| REQUIRED | *string* | A short, human-readable label to display for the source of the information displayed on this card. If a `url` is also specified, this MAY be the text for the hyperlink.
 `url` | OPTIONAL | *URL* | An optional absolute URL to load (via `GET`, in a browser context) when a user clicks on this link to learn more about the organization or data set that provided the information on this card. Note that this URL should not be used to supply a context-specific "drill-down" view of the information on this card. For that, use `link.url` instead.
 `icon` | OPTIONAL | *URL* | An absolute URL to an icon for the source of this card. The icon returned by this URL SHOULD be a 100x100 pixel PNG image without any transparent regions.
+`topic` | OPTIONAL | *[FHIR Coding](http://hl7.org/fhir/datatypes.html#coding)* | A "FHIR Coding" json object containing a descriptive topic related to the content of the card. A topic provides a high-level categorization that can be useful for filtering and searching.
+
+Below is an example `source` parameter:
+
+```json
+{
+  "source" : {
```

Comment:
Does the new field pertain to where the guidance comes from or what kind of guidance is provided?  Put another way, is this applicable to the source of the card's information or metadata for the card itself?  Couldn't a given source return different variations in decision support 'category'/'topic'?

Reply:
I don't think this should be added until we have a preferred value set.

Reply:
Created [jira](https://jira.hl7.org/browse/FHIR-25746) against Clinical Reasoning to create additional example codesystem and codes for DefinitionTopic.